src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -448,20 +458,40 @@ function PolicyAccountingPage({policy}: PolicyAccountingPageProps) {
                             titleStyles={styles.accountSettingsSectionTitle}
                             childrenStyles={styles.pt5}
                         >
-                            {connectionsMenuItems.map((menuItem) => (
-                                <OfflineWithFeedback
-                                    pendingAction={menuItem.pendingAction}
-                                    key={menuItem.title}
-                                    shouldDisableStrikeThrough
-                                >
-                                    <MenuItem
-                                        brickRoadIndicator={menuItem.brickRoadIndicator}
+                            {!(hasUnsupportedNDIntegration && hasSyncError) &&
+                                connectionsMenuItems.map((menuItem) => (
+                                    <OfflineWithFeedback
+                                        pendingAction={menuItem.pendingAction}
                                         key={menuItem.title}
-                                        // eslint-disable-next-line react/jsx-props-no-spreading
-                                        {...menuItem}
-                                    />
-                                </OfflineWithFeedback>
-                            ))}
+                                        shouldDisableStrikeThrough
+                                    >
+                                        <MenuItem
+                                            brickRoadIndicator={menuItem.brickRoadIndicator}
+                                            key={menuItem.title}
+                                            // eslint-disable-next-line react/jsx-props-no-spreading
+                                            {...menuItem}
+                                        />
+                                    </OfflineWithFeedback>
+                                ))}
+                            {hasUnsupportedNDIntegration && hasSyncError && (
+                                <FormHelpMessage
+                                    isError
+                                    shouldShowRedDotIndicator
+                                    style={styles.menuItemError}
+                                >
+                                    <Text style={[{color: theme.textError}]}>
+                                        {translate('workspace.accounting.errorODIntegration')}
+                                        <TextLink
+                                            onPress={() => {
+                                                // Go to Expensify Classic.
+                                                Link.openOldDotLink(CONST.OLDDOT_URLS.INBOX);
```

Comment:
Should we just take them directly to the accounting section of policy settings in OldDot? For example https://www.expensify.com/policy?param={"policyID":"C4193DA3E01F2257"}#connections? Seems like it would be more direct.